apps/console/src/features/claims/components/edit/local-claim/edit-basic-details-local-claims.tsx

```diff
@@ -200,6 +208,64 @@ export const EditBasicDetailsLocalClaims: FunctionComponent<EditBasicDetailsLoca
         }
     }, [ claim ]);
 
+    useEffect(() => {
+        getConnectorDetails(ServerConfigurationsConstants.USER_ONBOARDING_CONNECTOR_ID, 
+            ServerConfigurationsConstants.SELF_SIGN_UP_CONNECTOR_ID)
+            .then((response: GovernanceConnectorInterface) => {
+                setConnector(response);
+            })
+            .catch((error: AxiosError) => {
+                if (error?.response?.data?.detail) {
+                    dispatch(
+                        addAlert({
+                            description: t(
+                                "console:manage.features.governanceConnectors.notifications." +
+                                "getConnector.error.description",
+                                { description: error.response.data.description }
+                            ),
+                            level: AlertLevels.ERROR,
+                            message: t(
+                                "console:manage.features.governanceConnectors.notifications." +
+                                "getConnector.error.message"
+                            )
+                        })
+                    );
+                } else {
+                    // Display general error message.
+                    dispatch(
+                        addAlert({
+                            description: t(
+                                "console:manage.features.governanceConnectors.notifications." +
+                                "getConnector.genericError.description"
+                            ),
+                            level: AlertLevels.ERROR,
+                            message: t(
+                                "console:manage.features.governanceConnectors.notifications." +
+                                "getConnector.genericError.message"
+                            )
+                        })
+                    );
+                }
+            });
+    }, []);
+
+    useEffect(() => {
+ 
+        if (isEmpty(connector?.properties)) {
+            return;
+        }
+
+        connector.properties.map((property: ConnectorPropertyInterface) => {
+            if (property.name === ServerConfigurationsConstants.ACCOUNT_LOCK_ON_CREATION) {
+                if (property.value === "false") {
+                    setAccountVerificationEnabled(false);
+                } else {
```

Comment:
Since `property.value` is a string value, I think we cannot be 100% confident that this will always have either "true" or "false", can we? 

If so, shall we use a `else if` instead of `else` here?

```suggestion
                } else if (property.value === "true") {
```

Please correct me if I'm missing something :)

Reply:
We will have to add the same statement in the `else if` and `else` blocks in that case.